index.html

```diff
@@ -34,7 +34,13 @@
 <body>
     <div id="bg">
         <img class="bg" img src="./assets/images/bg.jpg" alt="">
+        <div id="myModal" class="modal">
+            <div class="modal-content">
+                <span class="close">&times;</span>
+                <p>Please select: restaurants, bars, or parks</p>
```

Comment:
```suggestion
                <p>Please select at least one: restaurants, bars, or parks</p>
```
adjust wording for modal